src/core/lib/security/credentials/jwt/jwt_credentials.h

```diff
@@ -21,6 +21,12 @@
 
 #include <grpc/support/port_platform.h>
 
+#include <string.h>
```

Comment:
But this should be string.
```suggestion
#include <string>
```